server/tests/src/main/scala/sttp/tapir/server/tests/ServerBasicTests.scala

```diff
@@ -63,6 +63,9 @@ class ServerBasicTests[F[_], ROUTE, B](
     testServer(endpoint, "POST empty endpoint")((_: Unit) => pureResult(().asRight[Unit])) { (backend, baseUri) =>
       basicRequest.post(baseUri).send(backend).map(_.body shouldBe Right(""))
     },
+    testServer(out_fixed_content_type_header, "Fixed content-type header")((_: Unit) => pureResult("".asRight[Unit])) { (backend, baseUri) =>
+      basicRequest.get(baseUri).send(backend).map(_.headers should contain only Header("Content-Type", "text/csv"))
```

Comment:
the content-length header is also returned, so I think you'll have to drop the `only` in this test